Gateway/Request/CreditCardBuilder.php

```diff
@@ -21,15 +22,24 @@ class CreditCardBuilder implements BuilderInterface
     public function build(array $buildSubject)
     {
         $payment = SubjectReader::readPayment($buildSubject);
-        $method  = $payment->getPayment();
+        $method = $payment->getPayment();
+
+        $paymentData = null;
```

Comment:
I think this one should be empty array than `null`

Reply:
it might be `null` IMO, because later there is only `setting` data into it. no other operations, this var is always `re-written`.

As a case in the past when there is later only `=` operation on var than it was set to `null` in the as declaration.